src/components/token/TokenRow.js

```diff
@@ -22,7 +26,48 @@ function TokenRow({ token }) {
     history.push(`/token_detail/${uid}`);
   };
 
-  return (
+  const Symbol = ({ children }) => {
+    return <div className="table-tokens-symbol">{children}</div>;
+  };
+
+  const renderIdCell = id => {
+    const idStart = id.substring(0, isMobile ? 4 : 12);
+    const idEnd = id.substring(id.length - (isMobile ? 4 : 12), id.length);
+
+    return (
+      <div className="id-cell">
+        {idStart}
+        <div className="ellipsis">
+          <div className="ellipsi"></div>
+          <div className="ellipsi"></div>
+          <div className="ellipsi"></div>
+        </div>
+        {idEnd}
+      </div>
+    );
+  };
```

Comment:
Didn't you create a component for this?
Should we use it here?

Reply:
Indeed. Fixed as sugested.